F21/gantlavamshikrishnareddy/vamshi_5030_03.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+"""Vamshi-5030-03.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1eHAIzmOG5Jj0v7Mlz2ogu-X_V-O3JNM6
+"""
+
+print("LANGUAGE CODE\nEnglish (en)\nTurkish (tr)\nAzerbaijani (az)\nIrish (ga)\nHindi  (hi)\nChinese   (zh)\nGreek  (el)\nJapanese  (ja)\nThai  (th)\nSpanish  (es)\nRussian  (ru)ETC...")
+lang=input(print("ENTER THE SELECTED LANGUAGE CODE : ", ))
+word=input(print("ENTER WORD FROM THE SELECTED LANGUAGE : ", ))
+print("THE GIVEN WORD IN LOWERCASE IS : ")
+
```

Comment:
Great! It seems you are trying to compare the the user input and the language. How about having a comment to describe this.